org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/DOMDocument.java

```diff
@@ -321,7 +321,7 @@ public boolean isBeforeProlog(int offset) {
 
 	private static SchemaLocation createSchemaLocation(DOMNode root, String schemaInstancePrefix) {
 		DOMAttr attr = root.getAttributeNode(getPrefixedName(schemaInstancePrefix, "schemaLocation"));
-		if (attr == null) {
+		if (attr == null || attr.getNodeAttrValue()==null) {

```

Comment:
Could you add a  moment with the two useczses.

Have uou à test with yiur new usecase?

Reply:
It is to check for this case: https://github.com/eclipse/lemminx/blob/8ea20b476980f02094ebecf85abd3c050ed7b5b2/org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/extensions/xsi/XSIFormatterExperimentalTest.java#L117